cmd/brew-portable-package.rb

```diff
@@ -1,28 +0,0 @@
-# Build a portable Homebrew utility bottle.
-#
-# Usage: brew portable-package <formula> [formula...]
-
-odie "Need to specify at least one portable formula!" if ARGV.empty?
-
-if RUBY_VERSION.split(".").first.to_i < 2
-  safe_system "brew", "install", "ruby"
-  ENV["HOMEBREW_RUBY_PATH"] = "#{HOMEBREW_PREFIX}/bin/ruby"
-end
-
-ENV["HOMEBREW_PREFER_64_BIT"] = "1"
```

Comment:
I'm saying `brew test-bot --portable ruby` should be sufficient if you want to remove this command.